charts/bctlquickstart/values.yaml

```diff
@@ -8,7 +8,7 @@ image:
   repository: bastionzero/bctl-quickstart
   pullPolicy: Always
   # Overrides the image tag whose default is the chart appVersion.
-  tag: "1.0.0" # For our quickstart job
+  tag: 1.1.0 # For our quickstart job
```

Comment:
For the future maybe we want to try to start using `latest` so we don't have to keep updating this value. But that change can be done at another time